src/Components/Shared/src/WebEventData.cs

```diff
@@ -243,5 +274,25 @@ private static ChangeEventArgs DeserializeChangeEventArgs(string eventArgsJson)
             }
             return changeArgs;
         }
+
+#nullable disable
```

Comment:
Why is this necessary?

Reply:
The file is shared between 3 projects which have different nullability enforcements. This causes issues with the source generated files. Turning it off seems like an easy workaround for the time being.

Reply:
Might be good to add that as a comment.